README.md

```diff
@@ -4,6 +4,10 @@
 
 Any distribution of the Java 17 JDK.
 
+For apple silicon you will want to enable the Docker option for `Use Rosetta for x86/amd64 emulation on Apple Silicon`.
+After enabling this option it is recommended that you delete all docker images and containers and rebuild them
+with this option enabled.
```

Comment:
Hmmm, I would put this under the `ReportStream Setup` heading since this setting is only required for ReportStream, not us.